pkg/web/wsHandler.go

```diff
@@ -0,0 +1,59 @@
+package web
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/gorilla/websocket"
+	"github.com/labstack/echo"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	writeWait  = 10 * time.Second
+	pongWait   = 60 * time.Second
+	pingPeriod = (pongWait * 9) / 10
```

Comment:
probably should consider making those configurable.

Reply:
ok, if we want to make this configurable, how best to go about it? optional command line arguments?

Reply:
yes, and environment variables